package.json

```diff
@@ -6,8 +6,8 @@
   "type": "module",
   "scripts": {
     "test": "npm run test:unit && npm run test:integration",
-    "test:unit": "jest --testPathIgnorePatterns=test/integration",
-    "test:integration": "jest --testPathPattern=test/integration --runInBand",
+    "test:unit": "jest --testPathIgnorePatterns=test/integration --coverage",
+    "test:integration": "jest --testPathPattern=test/integration --runInBand --coverage",
```

Comment:
a couple of things:

- if unit and integration tests, as I expect, have different levels of coverage, shall we not set them independently using cli flags here?
- on the other hand, if we want to use the same thresholds for both, why don't we move this --coverage flag to the config file instead?

Reply:
Good point, I will fix this.

Reply:
@simoneb Updated.